code42.json

```diff
@@ -1260,7 +1293,10 @@
                         "IN_PROGRESS"
                     ],
                     "default": "",
-                    "order": 1
+                    "order": 1,
+                    "contains": [
+                        "code42 alert state"
```

Comment:
The custom contains types should be prefixed with `code42`

Reply:
Doesn't `primary` need to be `true` on `alert_state` (and `username` above) for the contains values to be able to be used as inputs? Right now I can't take any actions on the alert state output from a search.

Reply:
I went through the rest of params that have any `contains` values, and the only other one I found missing `primary: true` is the `assignee` field on `list_cases`.